src/test/java/com/atlassian/bitbucket/jenkins/internal/config/BitbucketServerConfigurationTest.java

```diff
@@ -1,22 +1,23 @@
 package com.atlassian.bitbucket.jenkins.internal.config;
```

Comment:
Can you please add test cases for:
* The validate method
* The case where `creds` is `null` in `checkAdminCredentialsId`
* `base.getHost().contains("bitbucket.org")` in `checkBaseUrl` (not your code, I know) and any of the other uncovered methods wherever possible.